packages/bodiless-organisms/src/components/Menu/MenuContext.tsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright © 2020 Johnson & Johnson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, {
+  FC, createContext, useContext, useState, ComponentType, useEffect,
+} from 'react';
+import { observer } from 'mobx-react-lite';
+import { PageContextProvider, useEditContext, useUUID } from '@bodiless/core';
+
+type MenuContextType = {
+  isSubmenuOpen: boolean,
+  setSubmenuOpen: React.Dispatch<React.SetStateAction<boolean>>,
+};
+
+const MenuContext = createContext<MenuContextType>({
+  isSubmenuOpen: false,
+  setSubmenuOpen: () => null,
+});
+
+const useMenuContext = () => useContext(MenuContext);
+
+// Used for conditional fClasses.
+const isMenuOpen = () => {
+  const { isEdit } = useEditContext();
+  const { isSubmenuOpen } = useMenuContext();
+
+  return isEdit ? isSubmenuOpen : false;
+};
+
+const MenuProvider: FC = observer(({ children }) => {
```

Comment:
I don't think we need this context at all.  WIth the PageContextProvider, if we give it a type, we should be able to do something like
```
const useIsMenuOpen = () => {
  // Move up the context tree to see if we find an active menu context.
  for (let context = useEditContext();  context;  context = context.parent) {
    if (context.type == "menu" && context.isActive) return true;
  }.
  return false;
}
```

Reply:
Done